SampleTests/SampleTests.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>

```

Comment:
@srkirkland This means my samples will not be included, correct? I don't want them to be.

Reply:
@jSylvestre well it's a separate project and we aren't packing it anyway, so it doesn't matter